app/src/ui/lib/theme-change-monitor.ts

```diff
@@ -15,27 +19,26 @@ class ThemeChangeMonitor implements IDisposable {
   }
 
   private subscribe = () => {
-    if (!supportsDarkMode()) {
+    if (!supportsSystemThemeChanges()) {
       return
     }
 
-    remote.nativeTheme.addListener('updated', this.onThemeNotificationFromOS)
+    remote.nativeTheme.addListener('updated', this.onThemeNotificationUpdated)
   }
 
-  private onThemeNotificationFromOS = (event: string, userInfo: any) => {
-    const darkModeEnabled = isDarkModeEnabled()
-
-    const theme = darkModeEnabled
+  private onThemeNotificationUpdated = (event: string, userInfo: any) => {
+    const theme = getCurrentlyAppliedTheme()
       ? ApplicationTheme.Dark
       : ApplicationTheme.Light
```

Comment:
Shouldn't these two lines be removed entirely?

Reply:
Yes, not sure how I missed that 🤔

Reply:
Looking me, this worked because enum Dark (1) vs Light (0) became the proper true false for the statement.